cp-overrides.yaml

```diff
@@ -14,12 +14,16 @@ modules:
       git:
         url: https://github.com/jboss-container-images/jboss-eap-7-image.git
         ref: EAP_726_CR1
+
 osbs:
   configuration:
     container:
       platforms:
         only:
           - x86_64
+      compose:
```

Comment:
why do we need to override as it is the same values than `container.yaml`?

Reply:
When cekit runs with an override file present it will generate a new container.yaml that will be checked in to the specified dist git branch. Currently you cannot use container_file: and container: specifications at the same time, so they need to be fully specified if you override or change anything.